Code/ForceField/MMFF/testMMFFForceField.cpp

```diff
@@ -1439,7 +1440,7 @@ void testMMFFAllConstraints(){
   dc = new ForceFields::MMFF::DistanceConstraintContrib(field, 1, 3, 2.0, 2.0, 1.0e5);
   field->contribs().push_back(ForceFields::ContribPtr(dc));
   field->minimize();
-  TEST_ASSERT(MolTransforms::getBondLength(mol->getConformer(), 1, 3) > 1.99);
+  TEST_ASSERT(RDKit::round(MolTransforms::getBondLength(mol->getConformer(), 1, 3)) == 2);
```

Comment:
I'm going to go ahead and accept the PR, because this isn't a major thing, but the usual style for this kind of test is to use the feq() function instead of round()